lightningd/log.c

```diff
@@ -226,7 +228,8 @@ static void maybe_print(const struct log *log, const struct log_entry *l,
 			       l->io, tal_bytelen(l->io), log->lr->print_arg);
 }
 
-void logv(struct log *log, enum log_level level, const char *fmt, va_list ap)
+void logv(struct log *log, enum log_level level, bool notifer,
```

Comment:
`tell_notifier` is a better name for the bool I think?

Reply:
nice name!